File: backend/micro_epoweredmove_poi_chargingstation/src/main/java/com/kosdiam/epoweredmove/models/enums/EvType.java
package com.kosdiam.epoweredmove.models.enums;

public enum EvType {
    MHEV("Mild hybrid electric vehicle"),
    HEV("Hybrid electric vehicle"),
    PHEV("Plug-in hybrid electric vehicle"),
    BEV("Battery electric vehicle");

    private String evType;
    private EvType(String evType){
        this.evType = evType;
    }

    @Override
    public String toString(){
        return evType;
    }
}

File: frontend/E-PoweredMoveAndroidApp/app/src/main/java/com/kosdiam/epoweredmove/models/enums/EvType.kt
package com.kosdiam.epoweredmove.models.enums

import com.kosdiam.epoweredmove.R
import com.kosdiam.epoweredmove.models.interfaces.IEnumName

enum class EvType (override val stringResource: Int) : IEnumName {
    MHEV(R.string.ev_type_mhev),
    HEV(R.string.ev_type_hev),
    PHEV(R.string.ev_type_phev),
    BEV(R.string.ev_type_bev);
}